### DISCUSSION TITLE
Do i have to buy a license no matter which implementation i use?

### DISCUSSION THREAD

**Discussion by oliverunger** [phax/ph-schematron#172] [Q&A] [answered]
Related to the wiki stating:
_Schematron is part of the ISO 19757 standard "Information technology — Document Schema Definition Languages (DSDL)", Part 3 "Rule-based validation — Schematron". Until version 2016 the standard was free, since the 2020 version the standard needs to be bought._

https://github.com/phax/ph-schematron/wiki

**Comment by phax** [comment, marked as answer]
You only need to buy the Schematron specification from ISO if you want to read it. For the usage of the software you don't need it
